app/Http/Utilities/Correspondence.php

```diff
@@ -14,72 +14,80 @@ class Correspondence
-            'pro_tip' => null,
-        ],
-        [
-            'type' => 'reminder',
-            'key' => '5',
-            'label' => 'Last minute reminder v2',
-            'subject' => 'The Competition Deadline is Now!',
-            'body' => "You have until tonight at 10pm EST to upload your final, badass selfie with all of your [Report Back Noun]. \n\rUpload your final selfie with your [Report Back Noun] below. Can’t wait to see how you did, [First Name]!",
+            'label' => 'Last minute reminder',
+            'subject' => 'LAST CHANCE! The :campaign_name competition Deadline is now!',
+            'body' => ":first_name, \n\rThe :campaign_name Competition closes at 10:00 PM EST TONIGHT. Take this moment to click the icon below and upload your final photo clearly showing you and all :reportback_noun :reportback_verb. \n\rI will send the final results tomorrow, so keep your fingers crossed. Thanks for doing an awesome job to make an impact! \n\r:staff_name",
             'pro_tip' => null,
         ],
         [
             'type' => 'leaderboard',
             'key' => '1',
             'label' => 'Leaderboard update',
-            'subject' => '1st Leaderboard Update',
-            'body' => "Hello Competitors, \n\rHere is your 1st official [Campaign Name] competition update! The competition ends on [Competition End Date]. Finish in the top 3 in [Report Back Noun] [Report Back Verb] and win: \n\r- 1st place ­ $100 amex card\r- 2nd place ­ $50 amex card \r- 3rd place ­ $25 amex card \n\rKeep collecting [Report Back Noun] for the next 2 weeks to move up the leaderboard! \n\rPro tip ­ [Contest Pro Tip] \n\rNext update selfie with your [Report Back Noun] will be due [Leaderboard Day ­ 1] before 10pm est. \n\rHere is the leaderboard! Top 3 and a shoutouts below: \n\r[Leaderboard] \n\r[1 Random Image w/ comment] \n\r[Top 3 competitor IMAGES w/comment] \n\r[#] more weeks to make your mark and climb up the leaderboard!",
+            'subject' => '1st :campaign_name Leaderboard Update',
+            'body' => "Hello Competitors, \n\rHere is your first official :campaign_name competition update! The competition ends on :end_date. Finish in the top 3 in :reportback_noun :reportback_verb and win: \n\r- 1st place: ­ $100 amex card\r- 2nd place: ­ $50 amex card \r- 3rd place: ­ $25 amex card \n\rTwo more weeks to increase your number of :reportback_noun :reportback_verb to move up the leaderboard! \n\rPro tip ­- :pro_tip \n\rNext :reportback_noun will be due :leaderboard_msg_day-1 before 10pm est. \n\rHere is the leaderboard! Top 3 and a shoutouts below: \n\r[Leaderboard] \n\r[1 Random Image w/ comment] \n\r[Top 3 competitor IMAGES w/comment] \n\r[#] more weeks to make your mark and climb up the leaderboard!",
```

Comment:
let's remove the [Leaderboard] stuff from the default message till we have a solution for how we want to build this out